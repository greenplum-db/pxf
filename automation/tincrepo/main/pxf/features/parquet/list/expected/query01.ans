-- start_ignore
-- end_ignore
-- @description query01 for Parquet List data types (except timestamp List)
-- TODO: generate Parquet data on fly
 id |  bool_arr  |   smallint_arr   |         int_arr          |          bigint_arr_arr          |      real_arr       |     double_arr      |                                  text_arr                                   |         bytea_arr         |               char_arr                |                 varchar_arr           |                          date_arr                         |  numeric_arr  |
----+------------+------------------+--------------------------+----------------------------------+----------------------+--------------------+-----------------------------------------------------------------------------+---------------------------+---------------------------------------+---------------------------------------+-----------------------------------------------------------+
  1 | {}         | {50}             | {1}                      | {1}                              | NIL                  | {1.7e+308}         | {"this is a test string"}                                                   | NIL                       | {"hello          "}                   | {hello}                               | {2022-10-07 +00:00}                                       |
  2 | {f,t,t,f}  | {}               | {2,3}                    | NIL                              | {}                   | {1}                | {"this is a string with \"special\" characters","this is a string without"} | {}                        | {"this is exactly"," fifteen chars."} | {"this is exactly"," fifteen chars."} | {2022-10-07 +00:00}                                       |
  3 | {t}        | {-128}           | NIL                      | {}                               | {-123457,9.0072e+15} | {5.678,9.10234}    | {hello,"the next element is a string that says null","null"}                | {"\\xdeadbeef"}           | {"               "}                   | {""}                                  | {2022-10-07 +00:00, 2022-10-07 +00:00}                    |
  4 | NIL        | {10,20}          | {7,NULL,8}               | {-9223372036854775808,0}         | {2.3,4.5}            | NIL                | {NULL,""}                                                                   | {NULL,"\\x5c22"}          | NIL                                   | NIL                                   | {2022-10-07 +00:00, 2022-10-07 +00:00, 2022-10-07 +00:00} |
  5 | {t,f}      | NIL              | {}                       | {NULL,9223372036854775807}       | {6.7,-8,NULL}        | {}                 | NIL                                                                         | {"\\x5c5c5c",NULL}        | {"specials \\ \"   "}                 | {"specials \\ \""}                    | {}                                                        |
  6 | {t,f,NULL} | {0,127,-128}     | {2147483647,-2147483648} | {1,NULL,300}                     | {1e-14}              | {NULL,8.431,-1.56} | {"this is a test string with \\ and \"",NULL}                               | {"\\x313233","\\x343536"} | {"test string    ",NULL}              | {"test string",NULL}                  | {2022-10-07 +00:00, 2012-01-01 +00:00}                    |