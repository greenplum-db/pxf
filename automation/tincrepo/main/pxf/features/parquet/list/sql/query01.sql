-- @description query01 for Parquet List data types (except timestamp List)
SET bytea_output=hex;
SET datestyle TO 'ISO, MDY';

\pset null 'NIL'

SELECT * FROM pxf_parquet_list_types ORDER BY id;
