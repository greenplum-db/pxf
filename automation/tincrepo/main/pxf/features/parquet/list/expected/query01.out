-- start_ignore
-- end_ignore
-- @description query01 for Parquet List data types (except timestamp List)
SET bytea_output=hex;
SET
\pset null 'NIL'
Null display is "NIL".
SELECT * FROM pxf_parquet_list_types ORDER BY id;
 id |  bool_arr  | smallint_arr |         int_arr          |         bigint_arr         |       real_arr       |     double_arr     |                                  text_arr                                   |         bytea_arr         |               char_arr                |              varchar_arr              |           date_arr           |      numeric_arr
----+------------+--------------+--------------------------+----------------------------+----------------------+--------------------+-----------------------------------------------------------------------------+---------------------------+---------------------------------------+---------------------------------------+------------------------------+------------------------
  1 | NIL        | {50}         | {1}                      | {1}                        | NIL                  | {1.7e+308}         | {"this is a test string"}                                                   | NIL                       | {"hello          "}                   | {hello}                               | {2022-10-07}                 | {1.234500000000000000}
  2 | {f,t,t,f}  | {}           | {2,3}                    | NIL                        | NIL                  | {1.1}              | {"this is a string with \"special\" characters","this is a string without"} | {}                        | {"this is exactly"," fifteen chars."} | {"this is exactly"," fifteen chars."} | {2022-10-07}                 | {1.234500000000000000}
  3 | {t}        | {-128}       | NIL                      | {}                         | {-123457,9.0072e+15} | {5.678,9.10234}    | {hello,"the next element is a string that says null",NULL}                | {"\\xdeadbeef"}           | {"               "}                   | NIL                                   | {2022-10-07,2022-10-07}      | {1.234500000000000000}
  4 | NIL        | {10,20}      | {7,NULL,8}               | {-9223372036854775808,0}   | {2.3,4.5}            | NIL                | {NULL,""}                                                                   | {NULL,"\\x5c22"}          | NIL                                   | NIL                                   | {2022-10-07,2022-10-07,NULL} | {1.234500000000000000}
  5 | {t,f}      | NIL          | {}                       | {NULL,9223372036854775807} | {6.7,-8,NULL}        | NIL                | NIL                                                                         | {"\\x5c5c5c",NULL}        | {"specials \\ \"   "}                 | {"specials \\ \""}                    | NIL                          | {1.234500000000000000}
  6 | {t,f,NULL} | {0,127,-128} | {2147483647,-2147483648} | {1,NULL,300}               | {1e-14}              | {NULL,8.431,-1.56} | {"this is a test string with \\ and \"",NULL}                               | {"\\x313233","\\x343536"} | {"test string    ",NULL}              | {"test string",NULL}                  | {2022-10-07,2012-01-01}      | {1.234500000000000000}
  (6 rows)